packages/alea-frontend/pages/api/get-section-info/[courseId].ts

```diff
@@ -101,9 +101,13 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
   const coverageData = getCoverageData()[courseId];
   if (coverageData?.length) addVideoInfo(allSections, coverageData);
   if (!processedSlidesJson) {
-    const filePath = process.env.PROCESSED_SLIDES_JSON_PATH;
-    processedSlidesJson = JSON.parse(readFileSync(filePath, 'utf-8'));
+    const filePath = `${process.env.VIDEO_TO_SLIDES_MAP_DIR}/${courseId}_processed_slides.json`;
+    if (existsSync(filePath)) {
+      processedSlidesJson = JSON.parse(readFileSync(filePath, 'utf-8'));
+    }
+    if (processedSlidesJson) {
+      addClipInfo(allSections, processedSlidesJson);
+    }
   }
-  addClipInfo(allSections, processedSlidesJson);
```

Comment:
Moving this inside the `if` block would cause issues on 2nd and subsequent calls